datadog-lambda.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'aws-xray-sdk', '~> 0.11.3'
   # We don't add this as a direct dependency, because it has
   # native modules that are difficult to package for lambda
-  spec.add_development_dependency 'ddtrace', '~>1.5.0'
+  spec.add_development_dependency 'ddtrace', '~>1.11.0'
```

Comment:
Is the intent here to pin it to the `1.11` series of `ddtrace`? The way it is in this PR, only `1.11.x` will be installed, `1.12` and higher won't.
https://guides.rubygems.org/patterns/#pessimistic-version-constraint

Reply:
Yes, we tend to pin onto minor versions of the tracers. Since we (serverless) currently don't have a proper system to measure packaging or overhead increase in `ddtrace` for Ruby, as compared to JavaScript. We have to be on a way more pessimist and conservative side.